openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1119,6 +1121,92 @@ Cache-Control: no-store
 }
 ```
 
+# Callback Endpoint {#callback_endpoint}
+
+This endpoint is used by the Wallet to notify the Credential Issuer whether a credential has been successfully received or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL.
```

Comment:
We might want to be clearer about whether "OPTIONAL" means that it is optional for the Credential Issuer, or if it also means that it is optional for the wallet to support.

I think there is an argument for saying that the wallet MUST call this endpoint if the issuer supports it and provides a callback_id.

Reply:
I'm not sure we can make it a MUST if it's optional on the issuer's side to support? Perhaps SHOULD? 

The issuer can use a timeout after which it can presume that it won't receive a notification (due to network or the wallet not supporting this feature) - it may then decide to e.g. prompt for the presentation of the just-issued credential to verify that it was correctly stored, instead of being able to act off the callback's contents.

Reply:
My suggestion was "the wallet MUST call this endpoint if the issuer supports it and provides a callback_id."

I don't think I follow why that wording is problematic. i.e. I'm saying that if the issuer asks the wallet to make a callback, the wallet must (attempt to) make that callback.

Reply:
```suggestion
This endpoint is used by the Wallet to notify the Credential Issuer whether a credential has been successfully received or not. The Credential Issuer needs to return `callback_id` in the Credential Response or a Batch Credential Response for the Wallet to be able to use this Endpoint. Support for this endpoint is OPTIONAL. The wallet MUST call this endpoint if the issuer supports it and provides a `callback_id`.
```